build/tests/behat/bootstrap/FeatureContext.php

```
@@ -27,31 +35,52 @@ public function __construct() {
   /**
    * Set default browser window size to maximum.
    *
-   * @BeforeScenario @drupal
+   * @BeforeScenario
    */
   public function maximizeWindow() {
+    $driver = $this->getSession()->getDriver();
+    if (!($driver instanceof Selenium2Driver)) {
+      return;
```

Would it make more sense to throw an exception? You are silently returning at the moment